serving/src/test/java/feast/serving/service/CachedSpecServiceTest.java

```diff
@@ -63,70 +60,60 @@
   public void setUp() {
     initMocks(this);
 
-    store =
-        Store.newBuilder()
-            .setName("SERVING")
-            .setType(StoreType.REDIS)
-            .setRedisConfig(RedisConfig.newBuilder().setHost("localhost").setPort(6379))
-            .addSubscriptions(
-                Subscription.newBuilder().setProject("project").setName("fs1").build())
-            .addSubscriptions(
-                Subscription.newBuilder().setProject("project").setName("fs2").build())
-            .build();
-
-    when(coreService.registerStore(store)).thenReturn(store);
+    this.store = Store.newBuilder().build();
+    this.featureSetSpecs = new HashMap<>();
```

Comment:
Was the change from `LinkedHashMap` to `HashMap` intentional? The one is ordered.

Reply:
The original never used the ordered trait of `LinkedHashMap`, only calling `.get()` on it, hence the change to `HashMap`.